### PR TITLE
Feature/webpack

### PR DESCRIPTION
**Pull request by pink-bunny**

### CODE REVIEW

File: config/webpack.base.config.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          rules: [
+            {
+              test: /\.js$/,
+              exclude: /node_modules/,
+              use: {
+                loader: 'babel-loader'
+              }
+            },
+            {
+              test: /\.scss$/,
```
Comment (plzen) [inline config/webpack.base.config.js]: Add rules for images to import them like 
```
import Logo from “./logo.png”;
```

File: config/webpack.base.config.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+              ]
+            }
+          ]
+        },
+        plugins: [
+          new HtmlWebpackPlugin({
+            template: './src/index.html',
+            filename: './index.html'
+          }),
+          new CopyWebpackPlugin([ { from: 'src/static' } ]),
```
Comment (plzen) [inline config/webpack.base.config.js]: Do not use this plugin if you configure rules for images

Comment (pink-bunny) [reply config/webpack.base.config.js]: Done. Removed CopyWebpackPlugin

File: config/webpack.base.config.js (hunk 1 above)
Comment (pink-bunny) [reply config/webpack.base.config.js]: Done. Add image loader.

**Pull request merged**